mep/accounts/management/commands/export_events.py

```diff
@@ -172,7 +172,8 @@ def item_info(self, event):
     def source_info(self, footnote):
         '''source details from a footnote'''
         source_info = OrderedDict([
-            ('citation', footnote.bibliography.bibliographic_note)
+            ('citation', footnote.bibliography.bibliographic_note),
+            ('source_type', footnote.bibliography.source_type.name)
```

Comment:
This is an ordered dict so we can control the json output order, and I would like it to match the csv field order — so please put source type before citation.

Reply:
Makes sense! fixed in f488806